content/articles/auto-typing-js/index.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /how-to-handle-navigation-in-flutter/
+title: How to Handle Navigation in Flutter
+description: This article will show you how to navigate between different pages in Flutter. We will be building a simple app that uses an organized Navigation Named route.
+author: 
+date: 2021-06-01T00:00:00-18:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/how-to-handle-navigation-in-flutter/hero.jpg
+    alt: Handling Navigation in Flutter
+---
+In some cases, a developer may want to animate text content for users to see when they visit the web page. This can be achieved through auto-typing. This feature helps to enhance the look of a site and improves the overall user experience.
+<!--more-->
+Auto typing can be used to make eye-catching adverts or auto-complete text when a user is typing. It can be regarded as an animation and helps to retain attention, as well as simplify complex information. This is indeed a useful feature to implement in a frontend web project.
+
+### Prerequisites
+- A basic understanding of JavaScript.
+- An understanding of HTML and CSS is also an added advantage.
+
+### Goal
+In this article, we will implement the auto typing feature using JavaScript and [jQuery](https://jquery.com/).
+
+### Getting started
+We will use jQuery and [Immediately Invoked Function Expressions](https://www.javascripttutorial.net/javascript-immediately-invoked-function-expression-iife/) because they can be implemented quickly.
+
+The general syntax of an IIFE is shown below:
+
+```js
+(function () {
+  statements;
+
+})();
+
+```
+
+In the above example, the grouping operator prevents variables within the IIFE from being accessed from the global scope. 
+
+> You can learn more about IIFEs from official documentation found [here](https://developer.mozilla.org/en-US/docs/Glossary/IIFE). 
+
```

Comment:
You did not cover the get started portion adequately. You need to tell the reader to create a project folder, then add a JavaScript file rather than jumping straight to the code below.